.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish
```

Comment:
Copy this from some existing project, it misses the slack notification steps

Reply:
👍  Added slack support in 977f286